fake.py

```diff
@@ -232,7 +236,61 @@ def compose_frame(self, frame):
             if self.old_mask is None:
                 self.old_mask = mask
             mask = cv2.accumulateWeighted(mask, self.old_mask, self.MRAR)
+        # add a blue tint
+        holo = cv2.applyColorMap(img, cv2.COLORMAP_WINTER)
+        # add a halftone effect
+        bandLength, bandGap = 3, 4
+        for y in range(holo.shape[0]):
+            if y % (bandLength+bandGap) < bandLength:
+                holo[y,:,:] = holo[y,:,:] * np.random.uniform(0.1, 0.3)
+        # add some ghosting
+        holo_blur = cv2.addWeighted(holo, 0.2, self.shift_image(holo.copy(), 5, 5), 0.8, 0)
+        holo_blur = cv2.addWeighted(holo_blur, 0.4, self.shift_image(holo.copy(), -5, -5), 0.6, 0)
+        # combine with the original color, oversaturated
+        out = cv2.addWeighted(img, 0.5, holo_blur, 0.6, 0)
+        return out
+
+    def paddedzoom(self, image, scale=150):
+    
+        '''
+        Zoom in/out an image while keeping the input image shape.
+        i.e., zero pad when factor<1, clip out when factor>1.
+        there is another version below (paddedzoom2)
```

Comment:
Where is your second version? If you have a second version, perhaps consider to use a parameter to switch the behaviour?